templates/web/base/admin/triage/_inspect.html

```diff
@@ -1,3 +1,20 @@
+[% BLOCK category_list %]
```

Comment:
Probably better to add passing a select_name to admin/report-category.html, defaulting to 'category', instead of duplicating the template here :)